PZ-3365 - Adding mobileAppCheckout object to Payment response (#638)
mollie/api-documentation: source/reference/v2/payments-api/get-payment.rst

            - The URL your customer should visit to make the payment. This is where you should redirect the
              consumer to.

              .. note:: You should use HTTP ``GET`` for the redirect to the checkout URL. Using HTTP ``POST`` for
                        redirection will cause issues with some payment methods or iDEAL issuers. Use HTTP status code
                        ``303 See Other`` to force an HTTP ``GET`` redirect.

              Recurring payments don't have a checkout URL.

          * - ``mobileAppCheckout``

              .. type:: URL object
                 :required: false

            - The deeplink URL to the app of the payment method. Currently only available for ``bancontact``.

>               .. warning:: You should check if your consumer has the required app on their mobile

Reviewer: I'm not sure if this should be `customer` instead of `consumer`. Are there any usages of the term anywhere else?

Author: Not sure. We use it both in the docs, all around

Reviewer 2: I would say "your customer"

>               .. warning:: You should check if your consumer is having the required app on their mobile
>                            phone before redirecting to this URL. Mobile operating systems will ignore
>                            this URL if they do not have the correct app in place.

Reviewer: ```suggestion
              .. warning:: You should check if your consumer has the required app on their mobile
                           device before redirecting to this URL. Mobile operating systems will ignore
                           the redirect to this URL if the correct app is not installed.
```

>                            the redirect to this URL if they do not have the correct app is not installed.

Reviewer: ```suggestion
                           the redirect to this URL if the correct app is not installed.
```
